### ISSUE TITLE
create_newcase failing on blues

### ISSUE THREAD

**Issue by dqwu** [ESMCI/cime#1187]
This is reproducible on latest master branch.
```
./create_newcase -mach blues -compiler intel -case test_X -compset X -res f19_g16
Compset longname is 2000_XATM_XLND_XICE_XOCN_XROF_XGLC_XWAV 
Compset specification file is /blues/gpfs/home/wuda/cime/driver_cpl/cime_config/config_compsets.xml
Pes     specification file is /blues/gpfs/home/wuda/cime/driver_cpl/cime_config/config_pes.xml
Pes setting: grid          is a%1.9x2.5_l%1.9x2.5_oi%gx1v6_r%r05_m%gx1v6_g%null_w%null 
Pes setting: compset       is 2000_XATM_XLND_XICE_XOCN_XROF_XGLC_XWAV 
Pes setting: tasks       is {'NTASKS_ATM': -1, 'NTASKS_ICE': -1, 'NTASKS_CPL': -1, 'NTASKS_LND': -1, 'NTASKS_WAV': -1, 'NTASKS_ROF': -1, 'NTASKS_OCN': -1, 'NTASKS_GLC': -1, 'NTASKS_ESP': -1} 
Pes other settings: {'comment': 'none'}
ERROR: Incorrect number of matches, 0, for nodename 'entry' and attrs '{'id': 'ARCHIVE_SPEC_FILE'}' in file '/blues/gpfs/home/wuda/cime/cime_config/acme/config_files.xml'
```
This bug is confirmed to be caused by PR #1182. More specifically, by commit 33dcf97 (break up config_archive into component specific files).

**Comment by jedwards4b** [comment]
Fixed in PR #1184

**Comment by dqwu** [comment]
@jedwards4b 
Thanks, I tested PR #1184 on blues and it worked.
